site/src/pages/TemplatePage/TemplatePageView.stories.tsx

```diff
@@ -19,7 +19,19 @@ Example.args = {
 export const SmallViewport = Template.bind({})
 SmallViewport.args = {
   template: Mocks.MockTemplate,
-  activeTemplateVersion: Mocks.MockTemplateVersion,
+  activeTemplateVersion: {
+    ...Mocks.MockTemplateVersion,
+    readme: `---
+name:Template test
+---
+## Instructions
+You can add instructions here
+
+[Some link info](https://coder.com)
+\`\`\`
+# This is a really long sentence to test that the code block wraps into a new line properly.
+\`\`\``,
+  },
```

Comment:
I really like the way you tested this with a visual story instead of trying to locate the styles in a unit test.